src/pudl/transform/ferc1.py

```diff
@@ -1270,7 +1270,7 @@ class Ferc1AbstractTableTransformer(AbstractTableTransformer):
     """
     xbrl_metadata: pd.DataFrame = pd.DataFrame()
     """Dataframe combining XBRL metadata for both instant and duration table columns."""
-    xbrl_calculations: pd.DataFrame = pd.DataFrame()
+    xbrl_calculations: pd.DataFrame | None = None
```

Comment:
ah yes. I changed this here because some tables truly have no calculations, which results in an empty df for `xbrl_calculations` so I wanted a distinction between "i haven't tried to build this yet" (None) vs "there are just no calcs so its an empty df"

Reply:
there is one `calculation_components.empty` in the reconcile table calcs transform function. This is valid there bc if there are no calcs... then there are no calcs to validate.